test/pkgbmark.jl

```diff
@@ -29,8 +29,8 @@ function test_pkgbmark()
     stats = judgement_results_to_dataframes(judgement)
     @info stats
     @test length(keys(stats)) == 2
-    @test :target ∈ keys(stats)
-    @test :baseline ∈ keys(stats)
+    @test :lu ∈ keys(stats)
+    @test :qr ∈ keys(stats)
     @test size(stats[:target]) == (10, 5)
     @test size(stats[:baseline]) == (10, 5)
```

Comment:
```suggestion
    @test size(stats[:lu]) == (10, 5)
    @test size(stats[:qr]) == (10, 5)
```
Do these need to change?

Reply:
Other things had to change. Should all be green lights now.